package.json

```diff
@@ -14,6 +14,11 @@
   "version": "6.67.0",
   "main": "dist/index.js",
   "module": "index.js",
+  "files": [
```

Comment:
Not totally sure how `npm` is grabbing the correct files currently without this, maybe it's using `module and `main` and anything that includes those files?  Figured adding this was clearer and safer

Reply:
Yep seems like it!

> Certain files are always included, regardless of settings:
package.json
README
LICENSE / LICENCE
The file in the "main" field